src/logic/operate.js

```diff
@@ -0,0 +1,22 @@
+import Big from 'big.js';
+
+const operate = (numberOne, numberTwo, operation) => {
+  const num1 = Big(numberOne);
+  const num2 = Big(numberTwo);
+  let total = 0;
+
+  if (operation === '-') {
+    total = num1.minus(num2);
+  } else if (operation === '+') {
+    total = num1.plus(num2);
+  } else if (operation === 'X') {
+    total = num1.times(num2);
+  } else if (operation === '÷') {
+    total = num1.div(num2);
+  } else if (operation === '%') {
+    total = num1.mod(num2);
```

Comment:
When it comes to `%` we only need one operand right?
example: click `10` then `%` = 0.01

example: `num1 = (num1 *= 0.01).toString();`